README.md

```diff
@@ -1,10 +1,10 @@
-# PROPKA 3.1
+# PROPKA 3.2
```

Comment:
I'd be inclined to leave the version number off here or just call it PROPKA 3 (to indicate that this is all PROPKA 3.1-based).

My preference would be to call it PROPKA 3 in these docs.

Reply:
Addressed in 6b9f01a7c3d85548644895bbfc2d6e3e246f6ea9